bedrock/privacy/templates/privacy/archive/websites-2013-08.html

```diff
@@ -8,98 +8,97 @@
-        <section>
-          <h3>SMS Text Messages</h3>
-          <p>Mozilla provides an easy way to request an SMS text message with a link to download Firefox for Android. The mobile phone number you provide will be used and stored only for the purpose of sending requested communications and providing abuse blocking.</p>
-        </section>
-      </section>
-      <section>
-        <h2>Social Networking Services</h2>
-        <p>Mozilla may interact with you through social networks to further our mission. When you interact with us at a third-party social network, such as Facebook, Twitter, or Google +, the network gives us the ability to access and store certain information from your profile for that social network.</p>
-        <p>The categories of information we can access will depend, in part, on the permissions you grant us and the privacy settings you have established on that social network. In general, we try to minimize the amount of data we receive about you on social services. We will handle any information we receive from these services in accordance with this privacy policy.</p>
-      </section>
-      <section>
-        <h2>Legal Process and Other Disclosures</h2>
-        <p>Consistent with our privacy commitments, we will scrutinize third party requests for information about you for compliance with the law, including those coming from governmental agencies or civil litigants.  We may access, use, preserve or disclose information about you only when we have a good faith belief that it is reasonably necessary to do so to satisfy the applicable law, regulation, legal process or lawful governmental request of any country, or to protect the rights, property or safety of Mozilla, its users or the public. We will provide notice of legal process or governmental requests unless prohibited to do so by law or the circumstances warrant otherwise.</p>
-      </section>
-      <section>
-        <h2>Privacy Policy Changes</h2>
-        <p>By using the websites, you agree to the terms and conditions of this privacy policy. If you do not agree to the terms and conditions of this privacy policy, please do not use the websites. Mozilla may change its Privacy Policy from time to time. Any and all changes will be reflected on this page. Substantive changes will also be announced through the standard mechanisms through which Mozilla communicates with the Mozilla community, including Mozilla's “announce” <a href="https://lists.mozilla.org/listinfo/announce">mailing list</a> and <a href="https://groups.google.com/group/mozilla.announce">newsgroup</a>. </p>
-      </section>
-      <section>
-        <h2>For More Information</h2>
-        <p>If you have questions about this privacy policy, please contact the Mozilla Foundation at privacy@mozilla.org or the Mozilla Corporation at privacy@mozilla.com.</p>
-      </section>
-    </div>
-  </article>
+{% block title %}Websites Privacy Policy{% endblock %}
```

Comment:
This should have the Mozilla theme to match the current Websites Privacy Policy.